### PR TITLE
SOC-1753 - Duplicated post on post list page when other user posted

### PR DESCRIPTION
**Pull request by rafalkalinski**
Added the pivot parameter for the service request

https://wikia-inc.atlassian.net/browse/SOC-1753

### CODE REVIEW

File: front/scripts/main/models/discussion-forum.js [hunk 1]
```diff
@@ -107,6 +108,7 @@ DiscussionForumModel.reopenClass({
 			success: (data) => {
 				const contributors = [],
 					posts = data._embedded['doc:threads'],
+					pivotId = posts[0].id,
```
Comment (bkoval) [inline front/scripts/main/models/discussion-forum.js]: What if there are no posts? Wouldn't this throw an uncaught?

File: front/scripts/main/models/discussion-forum.js [hunk 2]
```diff
@@ -122,6 +124,7 @@ DiscussionForumModel.reopenClass({
 				forumInstance.setProperties({
 					contributors,
 					name: data.name,
+					pivotId,
```
Comment (bkoval) [inline front/scripts/main/models/discussion-forum.js]: This breaks alphabetical order ;)

**Comment by bkoval** [comment]
LGTM :police_car:

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**